src/parameters/parameters_limiter.h

```diff
@@ -0,0 +1,49 @@
+#ifndef __PARAMETERS_LIMITER_H__
+#define __PARAMETERS_LIMITER_H__
+
+#include <deal.II/base/parameter_handler.h>
+
+namespace PHiLiP {
+namespace Parameters {
+/// Parameters related to the limiter
+class LimiterParam
+{
+public:
+    /// Flag to perform convergence analysis for Limiter Tests (ie. burgers_limiter, advection_limiter, low_density_2d)
+    bool use_OOA;
+
+    /// Limiter type to be applied on the solution.
+    enum LimiterType {
+        none,
+        maximum_principle,
+        positivity_preservingZhang2010,
+        positivity_preservingWang2012
+    };
+    LimiterType bound_preserving_limiter;
+
+    // Epsilon value for Positivity-Preserving Limiter
+    double pos_eps;
+
+    /// Flag for applying TVB Limiter
+    bool use_tvb_limiter;
+
+    /// Maximum delta_x for TVB Limiter
+    double tvb_h;
```

Comment:
verbose variable names would be more clear than say 'mathematical' variable names; could you rename some of these ?

Reply:
Renamed the variables `tvb_h` to `max_delta_x` and `tvb_M` to `tuning_parameter_for_each_state`.